### ISSUE TITLE
Access 403

### ISSUE THREAD

**Issue by jaikishantulswani** [vincentcox/bypass-firewalls-by-DNS-history#1]
Hello @vincentcox ,
Its not an issue but I am curious that by abusing the DNS history can we able to access the ip / domain which is throwing 403 ?

Thank You.

**Comment by vincentcox** [comment]
Yes, that can be a possibility. It depends on how the server is configured.

**Comment by jaikishantulswani** [comment]
so we should we have to add http header i.e X-Forwarded-For / X-Forwarded-Host during the request ?

**Comment by vincentcox** [comment]
If you use burp you can add the IP and hostname here:
![image](https://user-images.githubusercontent.com/9286611/50208317-2ea0f900-0371-11e9-9be6-4578d9e604b1.png)

If you want it system-wide, you need to edit your hostfile (`/etc/hosts` or `c:\Windows\System32\Drivers\etc\hosts`).

**Comment by jaikishantulswani** [comment]
@vincentcox Thank You :+1:

**Comment by jaikishantulswani** [comment]
Hello @vincentcox ,
Is there any other way to do this as I have to resolve a bunch of domains with status 403 ?
